openreview/openreview.py

```diff
@@ -301,14 +307,15 @@ def post_note(self, note):
         if not note.signatures: note.signatures = [self.signature]
         response = requests.post(self.notes_url, json = note.to_json(), headers = self.headers)
         response = self.__handle_response(response)
-        return Note.from_json(response.json())
+
+        return self.get_note(response.json()['id'])
```

Comment:
ok. So I should keep the line:

return Note.from_json(response.json())

is that right?